pages/_app.js

```diff
@@ -1,11 +1,130 @@
 import GlobalStyle from "../styles";
 import { goals, categoryColors } from "../lib/data.js";
```

Comment:
You should try to avoid mixing `import` statement paths. In most statements you are using the shorthand `@/`, for the sake of consistency I suggest to stick to it here, too:  
```suggestion
import GlobalStyle from "@/styles";
import { goals, categoryColors } from "@/lib/data.js";
```

Reply:
Hey Thomas. 
Thanks for your review☀️
I will adjust the paths as you suggested.